terraform/aws/modules/storage/main.tf

```diff
@@ -47,7 +47,7 @@ resource "aws_db_instance" "km_db" {
   allocated_storage         = 20
   engine                    = "postgres"
   engine_version            = "10.6"
-  instance_class            = "db.t3.medium"
+  instance_class            = "db.m5.xlarge"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure the standard security best practices configuration is followed for AWS RDS instances**
 
**Vulnerability**
The AWS RDS Instance configuration does not comply with security best practices configuration.
 
**Remediation**
Configure the AWS instances to follow certain AWS Best Practice recommendations. Some of these are: 
 1. Enable performance insights. 
 2. Ensure allocated storage is not less than 200. 
 3. Ensure max allocated storage is not less than 500. 
 4. Consider using 'db.m5.xlarge' instance class for all instances.

source: accurics-app